testsuite/MDAnalysisTests/utils/test_modelling.py

```diff
@@ -134,26 +125,31 @@ def test_capping_inmemory(self):
                            u.trajectory.ts.dimensions)
 
 
-class TestMerge(TestCase):
-    ext = "pdb"
-
-    def setUp(self):
-        u1 = MDAnalysis.Universe(merge_protein)
-        u2 = MDAnalysis.Universe(merge_ligand)
-        u3 = MDAnalysis.Universe(merge_water)
-        self.universes = [u1, u2, u3]
-
-        suffix = '.' + self.ext
-        self.tempdir = tempdir.TempDir()
-        self.outfile = os.path.join(self.tempdir.name, 'test' + suffix)
-
-    def tearDown(self):
-        for u in self.universes:
-            del u
-        del self.tempdir
-
-    def test_merge(self):
-        u1, u2, u3 = self.universes
+class TestMerge(object):
+    # ext = "pdb"
+    
+    @staticmethod
+    @pytest.fixture()
+    def u1():
```

Comment:
Since we're at it, could you rename `u1`, `u2`, and `u3` to `u_protein`, `u_ligand`, and `u_water`, respectively. The current names are not informative, and there is no way anybody will remember which of `u2` and `u3` has the ligand.